src/database/Connector.java

```diff
@@ -124,16 +124,44 @@ public boolean verifyAccount(String username, String password) {
             String passwordInDB = resultPassword.getString(1);
             // password does not match
             if (!passwordInDB.equals(password)) {
-              System.err.println("the password does not match");
+              //System.err.println("the password does not match");
               return false;
             }
-            System.out.println("the account exists");
+            //System.out.println("the account exists");
             return true;
           }
         }
       }
     } catch (SQLException e) {e.printStackTrace();return false;}
-    System.err.println("the account does not exist");
+    //System.err.println("the account does not exist");
+    return false;
+  }
+
+  /**
+   * verify if the account username input by users matches an account in the
+   * database
+   *
+   * @param username
+   * @return true if the account exists, false if the account does not exist
+   */
+  public synchronized boolean verifyAccountExists(String username) {
+
+    String sqlSelectPassword = "select password from " + tableName + " where username = ?";
```

Comment:
It seems sketchy to use a `select password` as a test of the user's existence. Is there something else you can use, something that isn't a security risk?

Formatting strings like this can often be stored as constants at the top of the class.